front/pages/api/w/[wId]/assistant/builder/templates/index.ts

```diff
@@ -40,13 +40,9 @@ async function handler(
         visibility: "published",
       });
 
-      const filteredTemplates = templates.filter((t) => {
-        return t.presetAction === "reply" || t.presetActions.length > 0;
-      });
-
       return res
         .status(200)
-        .json({ templates: filteredTemplates.map((t) => t.toListJSON()) });
+        .json({ templates: templates.map((t) => t.toListJSON()) });
```

Comment:
OK to remove the filter, I'll put the templates not ready to be displayed in "draft" before shipping.